apps/web/src/server/appeals/appeals.router.js

```diff
@@ -18,6 +18,62 @@ router.route('/appeals-list').get(async (req, response) => {
 	});
 });
 
+// Temporary placeholder for router being developed in BOAT-64
```

Comment:
I was wondering if it would help to cross-ref tickets with a TODO: BOAT-64... Easier to search

Reply:
I've updated this as suggested